### PR TITLE
Transferring test cases to Dasharo Security module

### PR DESCRIPTION
**Pull request by Pokisiekk**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-security/20R-bios-setup-password.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Dasharo compatibility: BIOS Setup password
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-security/20R-bios-setup-password.md:1]: @Pokisiekk, we have some inconsistency here. In our products documentation, we had the product: `UEFI Setup password`. I think, that in documentation we should use the same nomenclature

File: docs/unified-test-documentation/dasharo-security/20S-usb-stack.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# Dasharo Security: USB stack
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-security/20S-usb-stack.md:1]: @Pokisiekk, we have some inconsistency here. In our products documentation, we had the product: `USB stack enable/disable`. I think, that in documentation we should use the same nomenclature

File: docs/unified-test-documentation/dasharo-security/20T-network-boot-availability.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Dasharo Security: Network Boot availability
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-security/20T-network-boot-availability.md:1]: @Pokisiekk, we have some inconsistency here. In our products documentation, we had the product: `Network stack enable/disable`. I think, that in documentation we should use the same nomenclature

File: docs/variants/msi_z690/test-matrix.md [hunk 4]
```diff
@@ -75,6 +73,9 @@ Certification Procedure for both tested platforms:
 | 6.   | [BIOS lock support][BLS]                          | BLS           | All                                  |
 | 7.   | [SMM BIOS write protection][SMM]                  | SMM           | All                                  |
 | 8.   | [Early boot DMA protection][EDP]                  | EDP           | All                                  |
+| 9.   | [BIOS Setup password][PSW]                        | PSW           | All                                  |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/test-matrix.md:76]: The nomenclature should be changed in accordance with the `Product IDs` documentation.

File: mkdocs.yml [hunk 5]
```diff
@@ -341,6 +340,9 @@ nav:
               - 'BIOS lock support': unified-test-documentation/dasharo-security/20J-bios-lock-support.md
               - 'SMM BIOS write protection': unified-test-documentation/dasharo-security/20O-SMM-bios-write-protection.md
               - 'Early boot DMA protection': unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md
+              - 'BIOS Setup password': unified-test-documentation/dasharo-security/20R-bios-setup-password.md
```
Comment (sulewskiprzemyslaw) [inline mkdocs.yml:343]: The nomenclature should be changed in accordance with the `Product IDs` documentation.

Also link to the `Network Boot availability` is broken.

File: docs/unified-test-documentation/dasharo-security/20R-bios-setup-password.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-security/20R-bios-setup-password.md]: Renamed

File: docs/unified-test-documentation/dasharo-security/20T-network-boot-availability.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-security/20T-network-boot-availability.md]: Renamed

File: docs/unified-test-documentation/dasharo-security/20S-usb-stack.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-security/20S-usb-stack.md]: Renamed

File: docs/variants/msi_z690/test-matrix.md (hunk 4 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/test-matrix.md]: Changed

File: mkdocs.yml (hunk 5 above)
Comment (Pokisiekk) [reply mkdocs.yml]: Changed

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**